/**: # wildcard for the node name
  ros__parameters:
    pool_limits_x: [0.0, 2.0]  # in m
    pool_limits_y: [0.0, 4.0]  # in m
    cell_resolution: 0.025  # in m
    # # middle standard obstacle
    # obstacle_limits_x: [0.75, 1.25]  # in m
    # obstacle_limits_y: [1.9, 2.1]  # in m
    # standard obstacle smaller
    obstacle_limits_x: [0.8, 1.2]  # in m
    obstacle_limits_y: [1.9, 2.1]  # in m